rpc/block_test.go

```diff
@@ -655,5 +655,4 @@ func validatePendingBlockHeader(t *testing.T, pBlock *PendingBlockHeader) {
 	require.NotZero(t, pBlock.L1GasPrice)
 	require.NotZero(t, pBlock.StarknetVersion)
 	require.NotZero(t, pBlock.L1DataGasPrice)
-	require.NotZero(t, pBlock.L1DAMode)
```

Comment:
This was failing because some blocks actually had "BLOB" (zero value of L1DAMode) as the `L1DAMode`, it's a valid option

Reply:
Could we use 	require.NotNil(t, pBlock.L1DAMode)?

Reply:
Sounds good, I'll do it